contrib/automation_tests/orbit_manual_instrumentation.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright (c) 2021 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
+from test_cases.capture_window import Capture, CheckTimers, VerifyTracksDoNotExist, VerifyTracksExist, CollapseOrExpandAllTracks, FilterTracks
+from test_cases.symbols_tab import LoadSymbols
+"""Test Orbit's manual instrumentation.
+
+Before this script is run there needs to be a gamelet reserved and
+"OrbitTestManualInstrumentation 3 10 100000" has to be started. Note that the 
+command line arguments matter, the test expects three worker threads.
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test it needs
+to by run from 64 bit python.
+
+This automation script covers the following workflow:
+ - start Orbit
+ - connect to a gamelet
+ - select a process and load debug symbols
+ - take a capture without api activated, verify no manual instrumentation is reported
+ - and take a capture with api activated, verify all the manual instrumentation shows up
+ - take another capture without api activated, verify no manual instrumentation is reported
+"""
+
+
+def main(argv):
+    tracks = [
+        "DynamicName_0", "DynamicName_1", "DynamicName_2", "DynamicName_3", "double_var",
+        "float_var", "int64_var", "int_var", "uint64_var", "uint_var", "ORBIT_ASYNC_TASKS",
+        "ORBIT_START_ASYNC_TEST"
+    ]
+    test_cases = [
+        ConnectToStadiaInstance(),
+        FilterAndSelectFirstProcess(process_filter='OrbitTest'),
+        LoadSymbols(module_search_string="OrbitTest"),
+        # Take a capture without manual instrumentation and assert that no tracks show up.
+        Capture(manual_instrumentation=False),
+        VerifyTracksDoNotExist(track_names=tracks),
+        # Take a capture with manual instrumentation and check for the existence of the tracks and the timers in the thread tracks.
+        Capture(manual_instrumentation=True),
+        VerifyTracksExist(track_names=tracks),
```

Comment:
It's not obvious from the naming, but `VerifyTracksExist` only checks visible tracks. With the amount of tracks you're checking, it might easily happen that some of them are off-screen on lower-res screens. Same for `VerifyTracksDoNotExist`.

This was the same issue that broke the GPU tests. I'm wondering if we should change this back to report the offscreen elements after all since we can't access the scrollbar yet. Alternatively, this can be done by filtering by track name, but this is also a bit tedious and tests the filtering functionality in addition.

Reply:
I ran into a releated issue with `CheckTimers`. The tracks seem to exist always, but the timer pane is only present when the track is visible. Solved that by filtering, collapsing and limiting the number of worker threads.

So I think filtering is also an option.

Reply:
Discussed offline: `VerifyTracksExist` works for invisible tracks so this is fine.